hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -74,6 +74,20 @@ paper_trade_account_balance:
 - - DAI
   - 1000
 
+# Balance Limit Configurations
+balance_asset_limit:
```

Comment:
Do we need to add a list of exchanges here? 
Also for this config, some examples on how to set it would be nice.

Reply:
You are not really meant to update the TEMPLATE file. But I will include some examples nonetheless.

Reply:
The examples i meant were to be in commented code. This is so that anyone who wants to update this manually in their config file can see the examples and follow along.